readme.md

```diff
@@ -56,16 +52,18 @@ Select 1:
 
 Create a variable and store the string "pizza" in it
 
-```js
-answer goes here
+```
+var pizzaVariable = "pizza";
 ```
```

Comment:
what would a good variable be for `"pizza"`. This is super nip picky and your syntax is obviously fine. I would recommend getting in the habit of semantic variable naming.